code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -38,32 +34,27 @@
 	icon_state = "dogbed"
 	desc = "A comfy-looking dog bed. You can even strap your pet in, in case the gravity turns off."
 	anchored = 0
+	buildstackamount = 10
+	buildstacktype = /obj/item/stack/sheet/wood
 
 /obj/structure/stool/bed/dogbed/ian
 	name = "Ian's bed"
 	desc = "Ian's bed! Looks comfy."
 	anchored = 1
 
-/obj/structure/stool/bed/dogbed/attackby(obj/item/weapon/W, mob/user, params)
-	if(iswrench(W))
-		playsound(loc, W.usesound, 50, 1)
-		new /obj/item/stack/sheet/wood(loc, 10)
-		qdel(src)
-
 /obj/structure/stool/bed/alien
 	name = "resting contraption"
-	desc = "This looks similar to contraptions from earth. Could aliens be stealing our technology?"
+	desc = "This looks similar to contraptions from Earth. Could aliens be stealing our technology?"
 	icon_state = "abed"
 
 /obj/structure/stool/bed/proc/handle_rotation()
 	return
 
 /obj/structure/stool/bed/attack_animal(var/mob/living/simple_animal/M)//No more buckling hostile mobs to chairs to render them immobile forever
 	if(M.environment_smash)

```

Comment:
Ah, I hadn't closely examined this existing bit of code enough - assumed it handled if they were buckled only. I see now, making this change in the next commit.